### DISCUSSION TITLE
[Feature Request]: GPS Deadzone for property security

### DISCUSSION THREAD

**Discussion by zl3ag** [meshtastic/firmware#4348] [Ideas]
Please add the ability to specify a deadzone location and radius which stops the GPS coordinates being reported when inside that area.

This is a security feature for vehicle attached trackers so that people can't see all your cars parked up at home.

position.deadzone_lat
position.deadzone_long
position.deadzone_radius

Thanks for your consideration.

**Comment by thebentern** [comment]
Stay tuned. I have an idea to implement an extension to our channel position precision feature that's a little bit different than what you're specifically requesting, but I think will satisfy the use-case well.

**Reply by zl3ag** [reply]
Cheers!
